man/man8/zpool.8

```diff
@@ -470,6 +480,33 @@ configuration.
 .Pp
 The content of the cache devices is considered volatile, as is the case with
 other system caches.
+.Ss Special Allocation Class
+The allocations in the special class are dedicated to specific block types.
+By default this includes all metadata, the indirect blocks of user data, and
+any dedup data.  The class can also be provisioned to accept a limited
+percentage of small file data blocks.
+.Pp
+A pool must always have at least one general (non-specified) VDEV before
+other devices can be assigned to the special class. If the special class
+becomes full, then allocations intended for it will spill back into the
+normal class.
+.Pp
+Dedup data can be excluded from the special class by setting the
+.Sy zfs_ddt_data_is_special
+zfs module parameter to false (0).
+Likewise, the indirect blocks for user data can be excluded from the special
+class by setting the
+.Sy zfs_user_indirect_is_special
+zfs module parameter to false (0).
+.Pp
+Inclusion of small file blocks in the special class is opt-in. Each dataset
+can control the size of small file blocks allowed in the special class by
+setting the
+.Sy special_small_blocks
```

Comment:
This tunable name is inconsistent with the others.  Maybe 'zfs_small_blocks_special' would at least match the previous two, but for sorting it may be better to have:

    zfs_special_ddt_blocks
    zfs_special_indirect_blocks
    zfs_special_small_file_blocks

Reply:
You're confusing a module parameter with a dataset property. Both have different naming conventions.